config.yml.default

```diff
@@ -1,131 +1,131 @@
+    resign_score: -1000            # If the score is less than or equal to this value, the bot resigns (in cp).
     resign_for_egtb_minus_two: true # If true the bot will resign in positions where the online_egtb returns a wdl of -2.
-    resign_moves: 3          # How many moves in a row the score has to be below the resign value.
-    offer_draw_enabled: false
-    offer_draw_score: 0      # If the absolute value of the score is less than or equal to this value, the bot offers/accepts draw (in cp).
+    resign_moves: 3                # How many moves in a row the score has to be below the resign value.
+    offer_draw_enabled: true       # Whether or not the bot should offer/accept draw.
+    offer_draw_score: 0            # If the absolute value of the score is less than or equal to this value, the bot offers/accepts draw (in cp).
     offer_draw_for_egtb_zero: true # If true the bot will offer/accept draw in positions where the online_egtb returns a wdl of 0.
-    offer_draw_moves: 5      # How many moves in a row the absolute value of the score has to be below the draw value.
-    offer_draw_pieces: 10    # Only if the pieces on board are less than or equal to this value, the bot offers/accepts draw.
+    offer_draw_moves: 10           # How many moves in a row the absolute value of the score has to be below the draw value.
+    offer_draw_pieces: 10          # Only if the pieces on board are less than or equal to this value, the bot offers/accepts draw.
 
   online_moves:
-    max_out_of_book_moves: 10 # Stop using online opening books after they don't have a move for 'max_out_of_book_moves' positions. Doesn't apply to the online endgame tablebases.
-    max_retries: 2           # The maximum amount of retries when getting an online move.
+    max_out_of_book_moves: 10      # Stop using online opening books after they don't have a move for 'max_out_of_book_moves' positions. Doesn't apply to the online endgame tablebases.
+    max_retries: 2                 # The maximum amount of retries when getting an online move.
     chessdb_book:
       enabled: false
       min_time: 20
-      move_quality: "good"   # One of "all", "good", "best".
-      min_depth: 20          # Only for move_quality: "best".
+      move_quality: "good"         # One of "all", "good", "best".
+      min_depth: 20                # Only for move_quality: "best".
     lichess_cloud_analysis:
       enabled: false
       min_time: 20
-      move_quality: "good"   # One of "good", "best".
-      max_score_difference: 50 # Only for move_quality: "good". The maximum score difference (in cp) between the best move and the other moves.
+      move_quality: "good"         # One of "good", "best".
+      max_score_difference: 50     # Only for move_quality: "good". The maximum score difference (in cp) between the best move and the other moves.
       min_depth: 20
       min_knodes: 0
     online_egtb:
-      enabled: false
+      enabled: true
```

Comment:
I don't think it should be enabled by default.

Reply:
Disabled